tests/harvest/test_scenarios.py

```
@@ -292,11 +305,24 @@ def test_scenario_four(
+    1. [First assertion or condition being tested]
+    2. [Second assertion or condition being tested]
+    3. ...
+
+    Args:
+        httpd_server_base (str): [Description of this parameter]
+        store_info_sets: [Description of this parameter]
+    """
     assert httpd_server_base
     for store in store_info_sets:
+        log.debug("testing scenario four for %s", store)
         log.debug(f"testing scenario one for {store}")
```

Ruff

Logging statement uses f-string

(G004)